packages/sirius-web/frontend/sirius-web-application/package.json

```diff
@@ -47,7 +47,9 @@
     "@types/react": "18.3.3",
     "@types/react-router-dom": "5.3.3",
     "@xstate/react": "3.0.0",
+    "export-to-csv": "1.3.0",
     "graphql": "16.8.1",
```

Comment:
You don't have to add a dependency to sirius-components-tables ?

Reply:
yes it what i have done in packages/tables/frontend/sirius-components-tables/package.json